pystiche_papers/johnson_alahi_li_2016/_nst.py

```diff
@@ -34,6 +34,40 @@ def training(
         Callable[[int, Union[torch.Tensor, pystiche.LossDict], float, float], None]
     ] = None,
 ) -> nn.Module:
+    r"""Training a transformer for the NST.
+
+    Args:
+        content_image_loader: Content images used as input for the ``transformer``.
+        style_image: Style image on which the ``transformer`` should be trained. If the
+            input is an string, the style image is read from the images in
+            :func:`~pystiche_papers.johnson_alahi_li_2016._images`.
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper. If omitted,
+            defaults to ``impl_params``.
+        transformer: Transformer to be optimized. If omitted, the
+            :func:`~pystiche_papers.johnson_alahi_li_2016.transformer` is used.
+        criterion: Optimization criterion. If ``None``, the default
```

Comment:
Since `None` plays no other role here than a placeholder for an omitted value, we should make this more clear in the description. I think it is more helpful to say "If omitted, x happens" than "If None, x happens. Defaults to None". Please address this in every docstring.